IdentityCore/src/network/request/MSIDAADRefreshTokenGrantRequest.h

```diff
@@ -37,6 +37,24 @@
                               refreshToken:(nonnull NSString *)refreshToken
                                     claims:(nullable NSString *)claims
                            extraParameters:(nullable NSDictionary *)extraParameters
+                                     apiId:(NSInteger)apiId
+                              forceRefresh:(BOOL)forceRefresh
```

Comment:
does it mean we will need to pass all telemetry parameters here and make an update to all network requests when we add a telemetry field? What if we want a dynamic list of additional parameters?

Reply:
changed